app/src/main/java/org/covidwatch/android/data/backend/firebase/FireBaseContactEventPublisher.kt

```diff
@@ -0,0 +1,77 @@
+package org.covidwatch.android.data.backend.firebase
```

Comment:
nit: this file should be `Firebase` instead of `FireBase`

Reply:
Done.